backend/src/openbeheer/utils/gherkin_e2e.py

```diff
@@ -123,22 +264,49 @@ def go_to_informatieobjecttype_create_page(
                 + f"/OZ/{furl(catalogus.url).path.segments[-1]}/informatieobjecttypen/create"
             )
 
-        def click_on_button(self, page: Page, name: str = "") -> None:
+        def user_selects_tab(self, page: Page, name: str = "") -> None:
             kwargs = {}
             if name:
                 kwargs.update({"name": name})
+
+            page.wait_for_load_state("networkidle")
+            button = page.get_by_role("tab", **kwargs)
+            button.click()
+            page.wait_for_load_state("networkidle")
+
+        # Actions
+
+        def user_clicks_on_button(self, page: Page, name: str = "") -> None:
+            kwargs = {}
+            if name:
+                kwargs.update({"name": name})
+
+            page.wait_for_load_state("networkidle")
             button = page.get_by_role("button", **kwargs)
             button.click()
+            page.wait_for_load_state("networkidle")
 
-        def click_on_link(self, page: Page, name: str = "") -> None:
+        def user_clicks_on_link(self, page: Page, name: str = "") -> None:
             kwargs = {}
             if name:
                 kwargs.update({"name": name})
+
+            page.wait_for_load_state("networkidle")
+
             iot_link = page.get_by_role("link", **kwargs)
+            href = iot_link.first.get_attribute("href")
```

Comment:
Note: I think that clicking on "normal" `<a>` tags breaks the routing in the E2E tests. This is why I made the link in the informatieobjecttype page a `NavLink`.

Reply:
As discussed: figure out how to deal with links in general, but this check is ok here